CHANGELOG

```diff
@@ -1,4 +1,5 @@
 [v#.#.#] ([MMM’YY])
+  * Enlarge images on click
```

Comment:
We could probably be more specific about _which_ images will enlarge, and how they'll enlarge.
"Images imported into issues will now expand to fill the screen when clicked" kinda thing. *Sell it!*